src/emissions.py

```diff
@@ -488,7 +490,7 @@ def calculate_nox_from_fuel_consumption(
                     "prime_mover_code",
                     "generator_id",
                 ]
-            ].drop_duplicates()
+            ].drop_duplicates().to_string()
```

Comment:
How will this render a dataframe? Will it still be readable as a string?

Reply:
The dataframe will show up in a nice format with rows and columns aligned